contracts/standard/permission/ArbitrablePermissionList.sol

```diff
@@ -285,12 +289,21 @@ contract ArbitrablePermissionList is PermissionInterface, Arbitrable {
         require(item.disputed);
 
         if (_ruling == REGISTER) {
-            if (item.status == ItemStatus.Resubmitted || item.status == ItemStatus.Submitted)
-                item.submitter.send(item.balance); // Deliberate use of send in order to not block the contract in case of reverting fallback.
-            else
-                item.challenger.send(item.balance);
-
-            item.status = ItemStatus.Registered;
+            if (rechallengePossible && item.status==ItemStatus.Submitted) {
+                uint arbitratorCost = arbitrator.arbitrationCost(arbitratorExtraData);
```

Comment:
This could have changed. We'll need to cache it in the Item struct.

Reply:
Yeah, but you are giving the submitter what's left after paying the current arbitration cost, not the cost at the time the dispute was raised. If the arbitration cost lowers, the submitter is stealing ETH from the contract.

Reply:
If the arbitration cost lowers, he effectively get back some of the ETH he has deposited for this purpose.

Reply:
Yeah you are right, I wasn't looking at how the balance is already updated from `challengeRegistration` at that point.